packages/core/src/lib/Select/OldSelect/OldSelect.styles.ts

```diff
@@ -0,0 +1,237 @@
+import { LabelProps, SelectedItemProps, ISelectProps } from '../types';
```

Comment:
i just noticed this is OldSelect after leaving comments, so this is not your code, right?

Reply:
yup this is just the old component
have kept it for now. till we finalize the new version 😄
resolved all conversation that are on old component file 🙌